Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewLayout.cs

```diff
@@ -155,7 +157,10 @@ protected void DetermineCellSize()
 			{
 				EstimatedItemSize = measure;
 			}
+		}
 
+		internal void FinishDetermineCellSize()
+		{
```

Comment:
Here we have split the size calculation into 2 methods, but we never seem to call this. There is a possible connection and type on `FinishedDeterminingCellSize` in `CollectionViewController.GetItemsCount`, but this appears to be for just 1 type. How are these two methods supposed to be used?

If there are 2 methods that need to be called, do we have a way to test that both are bing called? For me, the split gets scary as we now have forced a second method that can be missed. And by looking at the current code, it will only ever run on iOS when there is a SINGLE item in the source.

Reply:
Fixed the call, it should be always called, but you are correct that a better way will be preferred. Need to see what @hartez thinks about it.